### ISSUE TITLE
Adaptive Permutation testing - return number of permutations

### ISSUE THREAD

**Issue by iamsinht** [bhklab/wCI#7]
It would be helpful for any procedure which runs adaptive permutation testing to return the number of permutations that were run.  This would be useful both to gauge precision of p-values and to spot errors.

**Comment by p-smirnov** [comment]
@iamsinht agreed for spotting errors, although the precision is not easy to understand if you're using something like the QUICKSTOP algorithm.
